azavea/raster-foundry
# Feature/lk/default projects

## Overview
Add a `default_projects` user which we can add projects to in order to seed new user accounts
Add code to seed new user accounts

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [x] Symlinks from new migrations present or corrected for any new migrations~

## Notes
If the cloning queries are not successful, they will fail silently and not prevent the user from logging in.  The new projects probably won't show up right away (for instance, if you instantly nav to the projects view).

## Testing Instructions

* Add some projects with sensible settings / scenes to you database (you can manually copy them from an existing user and change the owner, created_by, and modified_by fields to the `default_projects` user
* Create a new user
* Log in using the user and verify that the projects were cloned successfully

Closes #1724
